doc/content/microreactors/gcmr/GCMR_Core_Neutronics.md

```diff
@@ -0,0 +1,100 @@
+| Lattice pitch (cm) | 20.8 |
+| Pin pitch (cm) | 2.0 |
+| TRISO fuel compact radius (cm) | 0.85 |
+| Moderator compact radius (cm) | 0.75 |
+| Cr coating thickness (cm) | 0.007 |
+| FeCrAl envelope thickness (cm)| 0.05 |
+| Burnable poison compact radius (cm) | 0.25 |
+| Coolant compact radius (cm) | 0.6 |
+| Control compact radius (cm) | 0.95 |
+| Fuel | TRISO, 40% packing fraction |
+| Coolant | He |
+| Moderator (Coating, Envelope)| YH1.8 (Cr, FeCrAl) |
+| Burnable poison absorber | Gd2O3 particles, 25% packing fraction |
+| Control rod | B4C (96% B-10 enrichment) |
+
+## Mesh
+
+MOOSE's Reactor Module was applied to create the mesh structure for the entire core of the GC-MR reactor. This tool was particularly useful in addressing intricate modeling prerequisites for the microreactor core such as fuel assemblies exhibiting different designs, distinct axial loading configurations of the burnable absorbers within the core, and modeling intricacies of control drum designs. The transition of the mesh from a 2D to a 3D structure, along with the incorporation of distinct axial regions, was made notably straightforward by employing the FancyExtruderGenerator. The tool's adaptability became especially useful when altering the mesh element size to examine the trade-off between accuracy and computational cost through coarser mesh. The meshing tool computational demands remained reasonable, making it a cost-effective choice for mesh generation, given the available mesh generation resources. [Fig_5]  illustrates a depiction of the 3-D GCMR, detailing both the axial and radial discretization.
+
+!media media/gcmr/core/Fig_5.png
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id=Fig_5
+      caption= Detailed 3-D GCMR core mesh
+
+
+
+!listing microreactors/gcmr/core/MESH/Griffin_mesh.i
+
+
+
+## Griffin Model
+
+The process initiated with generating homogenized multi-group cross-sections using Serpent-2, which were then converted into XML-format cross-section file. Griffin utilizes the cross-sections in XML-format file in conjunction with the mesh file. The 3D whole-core mesh was constructed utilizing MOOSE’s Reactor module, ensuring meticulous consistency between geometric representations in the mesh file and Serpent-2. Griffin solves the neutron transport equation employing discontinuous finite element DFEM with SN transport and CMFD acceleration, utilizing on-the-fly coarse mesh generation for CMFD. Efforts were dedicated to simplifying the 3D whole-core GC-MR mesh to alleviate computational demands, with careful consideration to avoid excessive mesh sizes, particularly in specific regions like the radial reflector and control drum areas, to ensure proper convergence of DFEM-SN with CMFD. 
+
+
+
+!listing microreactors/gcmr/core/Neutronics/Griffin_steady_state.i
+
+
+## Run Commands
+The Griffin code is a component of the blue_crab code package, and we can execute the simulation as follows:
```

Comment:
This section makes it seem like the simulation can be run out of the box, but the mesh needs to be generated first since it is imported with FileMeshGenerator. I would describe mesh generation first and then the simulation. Alternatively, you can also run the simulation and mesh generation as one command too by removing the [Mesh] block in Griffin_stead_state.i

Reply:
Indeed, you're correct. I've revised the section to commence with a description of mesh generation first.